pkg/virtctl/create/clone/clone_test.go

```diff
@@ -37,6 +37,14 @@ const (
 	vmKind, vmApiGroup             = "VirtualMachine", "kubevirt.io"
 	snapshotKind, snapshotApiGroup = "VirtualMachineSnapshot", "snapshot.kubevirt.io"
 )
+const (
+	LabelFilters = iota
+	AnnotationsFilters
+	LabelAndAnnotationsFilters
+	TemplateLabelFilters
+	TemplateAnnotationsFilters
+	TemplateLabelAndAnnotationsFilters
+)
```

Comment:
These can be unexported
```suggestion
const (
	labelFilters = iota
	annotationsFilters
	labelAndAnnotationsFilters
	templateLabelFilters
	templateAnnotationsFilters
	templateLabelAndAnnotationsFilters
)
```

Thanks